index.html

```diff
@@ -13,10 +13,12 @@
 
 <body class="body" id="body">
   <header class="header">
-   
+
+  
       <a href="#" class="logo"><p>MALEMBI</p></a>
    
-   
+    <nav>
+
       <img src="Union.png" class="nav-bar"> 
 <nav>
       <ul class="navlist">
```

Comment:
- [ OPTIONAL ] Good job adding more than required for this milestone, kindly center the text to meet the Figma design:
![Screenshot from 2023-02-16 12-22-44](https://user-images.githubusercontent.com/97834160/219339011-b48fc20b-98c8-43f3-8696-ee884e5c35e9.png)